test/generateMongoidTypes.test.ts

```diff
@@ -0,0 +1,41 @@
+import { exec } from 'child_process';
+import fs from 'fs';
+import { mongoidPrimitiveTypes } from '../src/model/dataTypes/primitiveDataTypes';
+import parser from '../src/parser';
+
+describe('generateMongoidTypes', () => {
+
+  const modelDir = './generated/mongoid';
```

Comment:
let's make this directory more test-specific

Reply:
Done